### ISSUE TITLE
Trailer do Filme

### ISSUE THREAD

**Issue by jhonattas** [criarideias/patrocine-front#4]
Não esta estartando na primeira vez que o usuario entra no site

**Comment by criarideias2023** [comment]
exatamente, o usuario tem que praticamente interagir com o site pra rodar.

**Comment by rachzy** [comment]
Isso na verdade é uma função de segurança imbutido na maioria dos navegadores (incluindo o **Chrome** e, obviamente, o **Opera**, que roda o Chrome em seu core), que impedem que algum arquivo de mídia seja tocado automaticamente (mesmo que esteja com autoplay) caso o usuário não tenha tido qualquer interação com o site.

A única forma de permitir que um vídeo seja tocado automaticamente por meio da tag "autoplay" é adicionando a tag "muted", o que obviamente, fará com que o vídeo fique mutado.

Mais informação sobre isso em: https://stackoverflow.com/questions/34764876/html-5-video-autoplay-not-automatically-starting-in-chrome

**Comment by jhonattas** [comment]
a proposito, nao era pra ter um background abaixo do video como failback ???

**Comment by rachzy** [comment]
[@kmzf777](https://github.com/kmzf777)
